content/en/serverless/aws_lambda/installation/dotnet.md

```diff
@@ -313,6 +313,16 @@ To use the optimized extension, disable Application Security Management (ASM), C
 
 Enabling any of these features cause the extension to default back to the fully compatible older version of the extension. You can also force your extension to use the older version by setting `DD_EXTENSION_VERSION` to `compatibility`. Datadog encourages you to report any feedback or bugs by adding an [issue on GitHub][8] and tagging your issue with `version/next`.
 
+## Adding Custom Spans
+
+When using the [Datadog Lambda tracing layer for .NET](https://github.com/DataDog/dd-trace-dotnet-aws-lambda-layer), ensure that a second version of the .NET tracer is not also packaged with your application code. Add `ExcludeAssets` instruction to ensure this extra tracer is excluded.
```

Comment:
This is way better, yet we should make sure that the wording is clear.

We don't explain why they need to have a second tracer. I'd write something around the lines 
> Adding custom spans requires you to have the Datadog .NET tracer installed in your application, therefore, you might need to add package reference instruction to exclude the additional code during runtime when using Datadog's .NET Lambda tracing layer.

And then explain why this is necessary.

WDYT?

Reply:
Alright, updated again to hopefully address this.